### ISSUE TITLE
About the performance on the val set of SemanticKITTI and nuSences

### ISSUE THREAD

**Issue by dante0shy** [yanx27/2DPASS#34]
Hi, really thanks for the great work. I found my result on val set has a gap to the reported results, and I found the reason should be some tricks, like TTA, mentioned in the README. Is there any results with your vanilla network on the SemanticKITTI and nuSences? So that I can check whether my training setting and process are correct. 

My current result is 66% for 64 epochs on val set of SemanticKITTI.

**Comment by yanx27** [comment]
@dante0shy Hi, 66 mIoU is a normal result, and you can obtain reported 69.3 mIoU after TTA.  If you train the codes on nuScenes, a correct result should be around 78, and 80.5 after TTA.

**Comment by yanx27** [comment]
@justice-heroo I think [this](https://github.com/yanx27/2DPASS/issues/27) issue has already comprehensively replied your questions. Test-time augmentation is a common trick to boost the performance, and previous works (e.g. SPVCNN, Cylinder3D etc.) all use this trick at online benchmark (`The authors of this Repo used heavy "tricks" to "boost" the performance`). And we compare their results on the test set with the same strategy. For the results of validation set, it is only used to validate the effectiveness of our proposed components for an ablation study, where all performance are gained after TTA. You can check the improvement by training the model without 2D information.  I also show that there are many other works use TTA for ablation in [this](https://github.com/yanx27/2DPASS/issues/27) issue. (`they even applied "tricks" on the validation set and reported the "boosted" results in their paper`)

Anyway, if you still have problem to the results, you can reply [this](https://github.com/yanx27/2DPASS/issues/27) issue and refute my point of view, but not use an anonymous account to teach others what `basic sense of research integrity` is😄.
